server/conn.go

```diff
@@ -380,7 +381,9 @@ func (cc *clientConn) handleQuery(sql string) (err error) {
 	} else {
 		err = cc.writeOK()
 	}
-	log.Debugf("[TIME_QUERY] %v %s", time.Now().Sub(startTs), sql)
+	costTime := time.Now().Sub(startTs)
+	log.Debugf("[TIME_QUERY] %v %s", costTime, sql)
+	metrics.QueryMetrics(costTime)
```

Comment:
I will do it in the next PR. This PR is mainly for add prometheus metrics framework.